process.go

```diff
@@ -639,7 +639,7 @@ func saveImageToFitBytes(po *processingOptions, img *vipsImage) ([]byte, context
 	img.CopyMemory()
 
 	for {
-		result, cancel, err := img.Save(po.Format, quality, conf.StripMetadata)
+		result, cancel, err := img.Save(po.Format, quality, conf.StripMetadata && !po.KeepMeta)
```

Comment:
`conf.StripMetadata && !po.KeepMeta` gives us a wrong result in case both `conf.StripMetadata` and `po.KeepMeta` are falsey. The best way to fix this is to set `po.KeepMeta` default value from config and use only `!po.KeepMeta` here.